backend/src/routes/v2/model/accessRequest/postAccessRequest.ts

```diff
@@ -69,6 +71,12 @@ export const postAccessRequest = [
     const accessRequest = await createAccessRequest(req.user, modelId, body)
 
     await audit.onCreateAccessRequest(req, accessRequest)
```

Comment:
To simplify route methods and group similar things together, a dedicator service method could be used for all post request stuff? Things like auditing, uploading to document store, sending emails, webhooks.